ArgoProject/src/components/Health.h

```diff
@@ -0,0 +1,35 @@
+﻿#ifndef _HEALTH_H
+#define _HEALTH_H
+
+namespace app::comp
+{
+	struct Health
+	{
+	public: // Constructors/Destructor/Assignments
+		Health() = default;
+		~Health() = default;
+
+		Health(Health const &) = default;
+		Health & operator=(Health const &) = default;
+
+		Health(Health &&) = default;
+		Health & operator=(Health &&) = default;
+
+	public: // Public Static Functions
+	public: // Public Member Functions
+	public: // Public Static Variables
+	public: // Public Member Variables
+		int health;
+		bool alive = true;
```

Comment:
`bool alive` is not needed since if `health <= 0` than we already know if entity isn't alive.